CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
 # Changelog
 
 ## 1.6.1 -- Unreleased
+
+### Deprecation Warnings
+* `AwsCrypto.encryptString()` and `AwsCrypto.decryptString()` are now deprecated.
```

Comment:
Since this is a changelog, we should use the declarative.

```suggestion
* Deprecated `AwsCrypto.encryptString()` and `AwsCrypto.decryptString()`.
```